Delete user UI functionalities (#561)
kitspace/kitspace-v2: README.md

```console
docker-compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.e2e.yml up e2e
```
##

> > 🚧 **GITEA_ADMIN_TOKEN is needed because we have hid gitea and it's inaccessible to other containers!**

Reviewer: ```suggestion
> Note: The GITEA_ADMIN_TOKEN is needed because the Gitea service is hidden and inaccessible to other containers!
```
